Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/package.json

```diff
@@ -16,6 +16,7 @@
     "@dnnsoftware/dnn-react-common": "^2.0.56",
     "babel-loader": "8.0.4",
     "babel-plugin-transform-react-remove-prop-types": "0.4.20",
+    "create-react-class": "^15.6.3",
```

Comment:
Is this required, was there a build warning or something ?

Reply:
Yeap, there is a warning:

![image](https://user-images.githubusercontent.com/22524011/52050675-f852be00-2559-11e9-9582-389581879ec2.png)